react-view-model.js

```diff
@@ -55,3 +58,9 @@ module.exports = namespace.reactViewModel = function reactViewModel(displayName,
 
 	return App;
 };
+
+// Expose Component and helpers to named imports
+module.exports.Component = Component;
+module.exports.promise = promise;
```

Comment:
I think `promise` should be `Promise`, as it is a constructor.

Reply:
Also a good point, but then we would be "masking" the native `Promise` in the file this was imported into. How about ...?
`PromiseHelper`?
`PromiseMap`?
`ObservablePromise`?
`PromiseObs`?

...any better ideas?

Reply:
hmm. Also a good point. I would go with `ObservablePromise` or `PromiseHelper`.